tenancy-api-mapping/openapispecs/generated/amc-app-orch-deployment-app-deployment-manager-openapi.yaml

```diff
@@ -73,7 +69,6 @@ paths:
               schema:
                 $ref: '#/components/schemas/ListClustersResponse'
           description: OK
-      summary: ListClusters
```

Comment:
I'm not sure why these are gone, but they'd change the name of the autogenerated client code so we should keep them

Reply:
To auto-generate the specs it used `make generate-specs` is anything else needed?